.github/dependabot.yml

```diff
@@ -6,18 +6,6 @@ updates:
       interval: daily
     commit-message:
       prefix: "Deps (NPM)"
-  - package-ecosystem: npm
```

Comment:
Why are these being removed? Shouldn't there be new settings for new directories?

Reply:
Dependabot should be able to traverse our file structure to find relevant manifest files for NPM (`package.json`), Cargo (`Cargo.toml`) and GitHub Actions (.yml file in `.github/workflows`)

Reply:
Actually I'm mistaken, only the nightly version of dependabot does this 😔 I will update the config to have all the relevant paths

Reply:
@alanmarkoTrilitech updated